### DISCUSSION TITLE
Neue Hoymiles Produkte  - MIT-4000-8T bis MIT-5000-8T und Batteriespeicher HM-MS-A2

### DISCUSSION THREAD

**Discussion by spcqike** [hoylabs/OpenDTU-OnBattery#1060] [General]
Hab grad zufällig gesehen: 

Hoymiles bringt neue 3-phasige Wechselrichter auf den Markt mit 4.000W, 4.500W und 5.000W.

komischer Weise heißen sie "8T", haben aber nur 4 MPPT. Durch die hohe Eingangsspannung und MPPT Startspannung (Eingang bis 140V, Start ab 32V) schätze ich mal, dass "8T" in dem Fall "8 Paneel" bedeutet. (oder heißt das "T" schon immer, wie für wie viele Module der Wechselrichter gedacht ist?)

https://www.shinetech.de/wp-content/uploads/2024/06/shinetech_datenblatt_mit-400045005000-8t_de.pdf

dazu gibt es den MS-A2, ein einphasiger Ac-gekoppelter Batteriespeicher. Dieser wird bereits auf Youtube von diversen Beta-Testern gezeigt
https://www.shinetech-power.de/wp-content/uploads/2024/06/Shinetech_Datasheet_MS-A2_EU_EN.pdf

so rein informativ 😄

**Comment by Isengo1989** [comment]
25 Jahre Garantie sind schon eine Ansage. Gibt es schon Preis-Leaks?

Zmd. laut dem hier -> https://www.computerbild.de/artikel/cb-Haus-Garten-Energie-Hoymiles-stellt-neuen-Solar-Wechselrichter-vor-38561845.html

**Comment by Manos1966** [comment]
Sind sie NRF24L01+ ?
Ich lese Kommunikation: Sub-1G 🤔 

> Die Sub-1G-Drahtloslösung garantiert die stabile Kommunikation mit der
Hoymiles Gateway-DTU und ermöglicht die Überwachung auf Modulebene
sowie Remote-B&W auf der Hoymiles Überwachungsplattform S-Miles
Cloud.

**Reply by spcqike** [reply]
Ich vermute sie sind, wie die HMS/HMT, bei 868MHz (Sub 1G) und damit CMT2300A.

**Comment by rainer667** [comment]
Gibt es dazu  schon weitere Infos?
Mir würde auch ein TSUN TSOL-MP3000 reichen
Vielen Dank Rainer

**Comment by steffenostrowski-byte** [comment]
Hallo, gibt es schon etwas neues zwecks der hoymiles mit-5000-8t serie. Wird es da zeitnahe eine opendtu Lösung für geben, oder muss man doch aus die hoymiles dtu zurück greifen?

**Reply by spcqike** [reply]
Du hast die richtige Stelle, den richtigen Request, im upstream doch bereits gefunden. 

Wie schnell das nun alles geht, kann ich nicht sagen. Scheinbar braucht es mehr zuarbeit von der Community. 

Wenn du bereits einen MIT hast, kannst du dich gern daran beteiligen. Zur originalen DTU kannst du ja immer noch wechseln, falls  man keine sinnvolle Kommunikation herstellen kann.

**Comment by steffenostrowski-byte** [comment]
Aktuell bin ich noch in der Planung/  Vorbereitung, bzw. Beim Bestellen, sonst würde ich gerne unterstützen

**Comment by andreas70** [comment]
Halte die MITs in Verbindung mit den üblichen "48V" Akkus für eher ungeeignet. Wenn überhaupt kriegt man maximal aus dem 4000er die Maximalleistung. Sind halt für 2S Panels je Eingang und damit für die doppelte Eingangsspannung ausgelegt. 
Ein "96V" (32S LiFePo4) Akku wär da optimal, da schauts aber mit verfügbaren Ladereglern und BMS eher düster aus.
Würde also eher dazu raten HMS bzw. HMT vorzusehen. Mehrere parallel funzt ja inzwischen recht gut, hab selber 3x HMS2000 laufen auf die 3 Phasen verteilt.

**Reply by huste511** [reply]
Ähm ... was hat ein AC-Microwechselrichter mit 48V Akkus zu tun?

und btw: mit normalen 1/2 Cut-Modulen bekomt man auch kein Akku geladen, dafür müssen es schon 1/3 Cut sein.

**Reply by andreas70** [reply]
Du weisst schon daß Du hier bei bei OpenDTU-**OnBattery** bist und für was das taugt??
Geladen werden 48V Akkus natürlich mit 2..x S Strings + passendem Laderegler...

**Reply by huste511** [reply]
Ja, aber was Du da in Verbindung mit dem MIT-5000 bzw. einem 32S Akku vorhast, dass sehe ich (in der Größenordnung) nicht so ganz.

**Comment by huste511** [comment]
Schon was neues zu dem Hoymiles MIT-5000-8T?
Habe nicht gelesen dass er unterstützt wird aber auch nicht das Gegenteil.

Scheint mit für ein Dach mit 10kWp eine nette Option zu sein, wenn man nicht zu wieviele WR und den hoheren Eta durch die höhere Spannung sucht.

**Comment by LeoIsBack** [comment]
Hi, es wäre toll wenn der MIT-5000-8T unterstützt würde. Ich habe noch keinen, würde aber gerne einen für ein Upgrade meiner PV benutzen. Wird hieran noch gearbeitet?

**Reply by spcqike** [reply]
es gibt einen offenen PR im Upstream

https://github.com/tbnobody/OpenDTU/pull/2962
